src/main/java/de/szut/lf8_project/lf8/project/ProjectController.java

```diff
@@ -28,6 +28,8 @@
 public class ProjectController {
     private final ProjectService service;
     private final ProjectMapper mapper;
+    private final Map<Integer, ProjectController> projects = new HashMap<>();
+    private final Map<Integer, String> projectLeaders = new HashMap<>();
```

Comment:
private final Map<Integer, ProjectController> projects = new HashMap<>();
    private final Map<Integer, String> projectLeaders = new HashMap<>();

can also be deleted.

Reply:
```suggestion
```